### ISSUE TITLE
PhotoView App Causes Restart on TCL

### ISSUE THREAD

**Issue by Remorse6** [chtaylo2/Roku-GooglePhotos#483]
Running the most recent (updated this morning) version of PhotoView on a TCL 4K/HDR screen.  The app starts, loads and runs as expected for several minutes before the screen abruptly restarts while scrolling thru a photo album (mine and shared with me). Haven't tried videos yet...

**Comment by DieGlobalists** [comment]
Similar Roku TCL issue, happened sometime week of 12/16/24?

IF you set Photoview to an album and then preview it, Roku does a restart reboot, THEN can only display all your google photos instead of specific albums. 

So I tried Roku photo stream: 
IF you use Roku Stream (never used it before this issue) it can’t access my Google photos. After clicking create, it has a disclaimer page, x out of this and you get a FAST (fraction of a second) error message stating can’t verify? Too fast to screen capture and send to Roku support!

So I had daughter try to add Photoview to our 2 devices, but it couldn’t download her massive Google photo library. 

So I had her try Roku photo stream. It accessed her Google photos! We setup a shared google album for our family pics on all devices and we can all add to them!

Bottom line: 
Photoview has a very recent bug, will wait for fix.
I have a recent problem with Google photos.
We will use Roku photo stream until Photoview fix.
Hope this helps.

**Comment by chtaylo2** [comment]
See: https://github.com/chtaylo2/Roku-GooglePhotos/issues/587
